README.md

```diff
@@ -4,12 +4,13 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/tailscale/tailscale-client-go)](https://goreportcard.com/report/github.com/tailscale/tailscale-client-go)
 ![Github Actions](https://github.com/tailscale/tailscale-client-go/actions/workflows/ci.yml/badge.svg?branch=master)
 
+DEPRECATED - V1 is no longer being maintained. The [V2 SDK](v2) provides a full wrapper around the V2 [Tailscale API](https://tailscale.com/api).
```

Comment:
Is "full" here implying that we've got complete / full coverage of the V2 endpoints? If so we might want to drop it from this sentence since we don't quite cover the entire V2 API (e.g., user invites are not in the client as an example).